pkg/flowaggregator/podcontroller/pod_controller.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package podcontroller
+
+import (
+	"sync"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+
+	"k8s.io/client-go/informers"
+	coreinformers "k8s.io/client-go/informers/core/v1"
+	corelisters "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+)
+
+const (
+	controllerName = "FlowAggregatorPodController"
+	// Set resyncPeriod to 0 to disable resyncing.
+	resyncPeriod time.Duration = 0
+)
+
+type PodInfo struct {
+	Namespace string
+	Name      string
+	NodeName  string
+}
+
+type PodController struct {
+	podInformer     coreinformers.PodInformer
+	podLister       corelisters.PodLister
+	podListerSynced cache.InformerSynced
+	podIPStore      map[string]PodInfo
+	mutex           sync.Mutex
+}
+
+func NewPodController(
+	informerFactory informers.SharedInformerFactory,
+) *PodController {
+	podInformer := informerFactory.Core().V1().Pods()
+	controller := &PodController{
+		podInformer:     podInformer,
+		podLister:       podInformer.Lister(),
+		podListerSynced: podInformer.Informer().HasSynced,
+		podIPStore:      make(map[string]PodInfo),
+	}
+	podInformer.Informer().AddEventHandlerWithResyncPeriod(
+		cache.ResourceEventHandlerFuncs{
+			AddFunc:    controller.addPod,
+			UpdateFunc: controller.updatePod,
+			DeleteFunc: controller.deletePod,
+		},
+		resyncPeriod,
+	)
+	return controller
+}
+
+func (p *PodController) GetPodInfoByIP(podIP string) (PodInfo, bool) {
+	p.mutex.Lock()
+	defer p.mutex.Unlock()
+	podInfo, exist := p.podIPStore[podIP]
+	if !exist {
+		klog.V(4).Infof("No pod is cached with the corresponding ip %s.", podIP)
```

Comment:
```suggestion
		klog.V(4).Infof("No Pod is cached with the corresponding IP %s", podIP)
```
Please capitalize "IP" and remove all trailing periods in following logs.